src/test/java/uk/gov/hmcts/reform/civil/bpmn/CreateClaimTest.java

```diff
@@ -392,15 +409,17 @@ void shouldAbort_whenStartBusinessProcessThrowsAnError() {
     @Nested
     class PreFlowStateRename {
```

Comment:
I guess we should remove this nested class now, looks like we forgot about it, because there are no ongoing flow state renaming tickets